tensorboard/webapp/hparams/_redux/hparams_reducers.ts

```diff
@@ -87,7 +89,82 @@ const reducer: ActionReducer<HparamsState, Action> = createReducer(
         metrics: nextMetricFilters,
       },
     };
-  })
+  }),
+  on(actions.dashboardHparamColumnAdded, (state, {column, nextTo, side}) => {
+    const {dashboardDisplayedHparamColumns: oldColumns} = state;
+
+    let destinationIndex = oldColumns.length; // Default to append at end.
+    if (nextTo !== undefined && side !== undefined) {
+      const nextToIndex = oldColumns.findIndex(
+        (col) => col.name === nextTo.name
+      );
+      if (nextToIndex !== -1) {
+        destinationIndex = side === Side.RIGHT ? nextToIndex + 1 : nextToIndex;
+      }
+    }
+    let newColumn = {...column, enabled: true};
+    let newColumns = [...oldColumns];
```

Comment:
This is a little nitpicky but I definitely prefer to keep as variables `const` when possible.
```suggestion
    const newColumn = {...column, enabled: true};
    const newColumns = [...oldColumns];
```

Reply:
Oops forgot to convert these. Applied!